ansible/openhpc.yml

```diff
@@ -1,4 +1,8 @@
 ---
+# Add cluster nodes to SSH known hosts
+- import_playbook: ssh-known-hosts.yml
+  when: ssh_known_hosts_enable | default(false)
```

Comment:
Any reason not to add this to other appliances? k8s, swarm, etc.

Reply:
It's a good playbook.  Can we incorporate this into the top-level playbook for pulling in the cluster_setup role?

Reply:
Could do. The container-infra-configure.yml playbook doesn't include that so we'd need to add it there also.